lib/users.ts

```diff
@@ -46,35 +46,9 @@ export namespace users {
                 return response.data;
             });
 
-        list = (): Promise<[User]> =>
-            this.client.get<[User]>(this.localPath).then((response) => {
-                return response.data;
-            });
-
         get = (id: string): Promise<User> =>
             this.client.get<User>(this.localPath + '/' + id).then((response) => {
                 return response.data;
             });
```

Comment:
`Get` has also vanished.

Reply:
@lukasmalkmus thanks for review. It wasn't clear to me from reading the notion page, the explanation makes sense, I updated the code.